docs/rpc-endpoints.md

```diff
@@ -0,0 +1,233 @@
+# RPC Endpoints
+
+### POST /v2/transactions
+
+This endpoint is for posting _raw_ transaction data to the node's mempool.
+
+Rejections result in a 400 error.
+
+### GET /v2/accounts/[Principal]
+
+Get the account data for the provided principal.
+The principal string is either a Stacks address or a Contract identifier (e.g., 
+`SP31DA6FTSJX2WGTZ69SFY11BH51NZMB0ZW97B5P0.get-info`
+
+Returns JSON data in the form:
+
+```
+{
+ "balance": 100,
+ "nonce": 1,
+ "balanceProof": "01fa...",
+ "nonceProof": "01ab...",
+}
+```
+
+Where balance (a u128) and nonce (a u64) are both integers, and the proofs are provided as
+hex strings.
+
+For non-existent accounts, this _does not_ 404, rather it returns an
+object with balance and nonce of 0.
+
+### POST /v2/map_entry/[Stacks Address]/[Contract Name]/[Map Name]
```

Comment:
Can this be a `GET`?  If the user is going to submit human-readable data (i.e. the hex-encoded key), we might as well make it part of the request path.  Thoughts?

Reply:
I don't think so -- the key data could be up to 1MB (in memory representation), and then URL encoded would be expanded a bit more, and paths that long should probably be avoided.

Reply:
Ah, gotcha.